mobile/src/main/java/org/openhab/habdroid/ui/MainActivity.java

```diff
@@ -994,13 +1012,24 @@ private void launchVoiceRecognition() {
         try {
             startActivity(speechIntent);
         } catch (ActivityNotFoundException speechRecognizerNotFoundException) {
-            try {
-                startActivity(new Intent(Intent.ACTION_VIEW,
-                        Uri.parse("market://details?id=com.google.android.googlequicksearchbox")));
-            } catch (ActivityNotFoundException appStoreNotFoundException) {
-                Toasty.error(this, R.string.error_no_app_store_found,
-                        Toast.LENGTH_LONG, true).show();
-            }
+                mLastSnackbar = Snackbar.make(findViewById(android.R.id.content),
```

Comment:
Shouldn't you close previous snackbar (if present) here ...

Reply:
Done